app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -12,7 +12,17 @@
     <% end %>
   <% end %>
 <% end %>
-<%= presenter.attribute_to_html(:non_user_collections, render_as: :collection_link, html_dl: true, label: t("simple_form.labels.defaults.set")) %>
+<% docs = [] %>
+<% presenter.non_user_collections.each do |collection| %>
+  <% doc = SolrDocument.find(collection) %>
+  <% docs << doc if current_ability.current_user.can?(:read, doc)%>
+<% end %>
+<dt>Collections</dt>
+<dd><ul class="tabular">
+  <% docs.each do |doc| %>
+    <li class="attribute attribute-collections_label"><%= link_to doc.title_or_label, doc %><%= " (#{doc.visibility})" %></li><br>
```

Comment:
I'm not too excited about including the visibility, but maybe ping azle/julia with the images